astro/src/content/docs/get-started/core-concepts/tenants.mdx

```diff
@@ -162,6 +162,9 @@ Control Email and Phone Number identity verification settings from one place.
   <APIField name="Unverified behavior" optional since="1.99.9">
```

Comment:
a lot of these say since 1.99.9 but only the phone portion is new. not sure how to say that, maybe 1 single aside at the top? @bhalsey might be in best spot to tackle this with 2601 work?